openapi.json

```diff
@@ -266,6 +266,131 @@
         ]
       }
     },
+    "/clusters/reports/": {
+      "get": {
+        "summary": "Returns the latest reports for the given list of clusters.",
+        "operationId": "getReportsForClusters",
+        "description": "Reports that are going to be returned are specified by list of cluster IDs that is part of request body.",
+        "parameters": [
+          {
+            "name": "clusterIDs",
+            "in": "body",
+            "required": true,
+            "description": "List of cluster IDs. Each ID must conform to UUID format. An example: `34c3ecc5-624a-49a5-bab8-4fdc5e51a266.",
+            "schema": {
+              "type": "array",
+              "items": {
+                "type": "string",
+                "minLength": 36,
+                "maxLength": 36,
```

Comment:
@tisnik  Did you keep in mind that URL length is around 2000 chars? Therefore a requester should implement a sort of pagination if it needs a big list of clusters.

Reply:
we don't want to use URL for providing any parameters (for this particular call). Everything is sent via request body.

Reply:
What request body? Your endpoint is for GET that means all parameters go to URL.

Reply:
semantically it is GET, and the parameters are send in body. IIRC it is valid according to rfc7231 (the old http 1.1 is no longer valid) and that approach is used for example in ElasticSearch. We can change it to POST, of course, but then it will have wrong semantic.

(yes, old HTTP is broken in many way)